packages/json/src/browser/json-validation-registry.ts

```diff
@@ -0,0 +1,39 @@
+/********************************************************************************
+ * Copyright (c) 2020 SAP SE or an SAP affiliate company and others.
+ *
+ * This program and the accompanying materials are made available under the
+ * terms of the Eclipse Public License v. 2.0 which is available at
+ * http://www.eclipse.org/legal/epl-2.0.
+ *
+ * This Source Code may also be made available under the following Secondary
+ * Licenses when the conditions for such availability set forth in the Eclipse
+ * Public License v. 2.0 are satisfied: GNU General Public License, version 2
+ * with the GNU Classpath Exception which is available at
+ * https://www.gnu.org/software/classpath/license.html.
+ *
+ * SPDX-License-Identifier: EPL-2.0 OR GPL-2.0 WITH Classpath-exception-2.0
+ ********************************************************************************/
+
+import { injectable } from 'inversify';
+import { Disposable } from '@theia/core/lib/common/disposable';
+import { deepClone } from '@theia/core';
+import { JsonSchemaConfiguration } from '@theia/core/lib/browser/json-schema-store';
+
+@injectable()
+/**
+ * Holds JSON Validation contributions from plugins / VSCode extensions.
+ * -  https://code.visualstudio.com/api/references/contribution-points#contributes.jsonValidation
+ */
+export class JsonValidationContributionsRegistry {
+
+    protected readonly registry: JsonSchemaConfiguration[] = [];
+
+    registerJsonValidation(jsonValidationEntry: JsonSchemaConfiguration): Disposable {
+        this.registry.push(jsonValidationEntry);
+        return Disposable.NULL;
```

Comment:
`Disposable.NULL` is bogus, please use `JsonSchemaStore.registerSchema` instead. it has a proper logic to unregster configuration

Reply:
@bd82 It was not addressed you this remove entry from `this.registry` and json language server should be notified about it as well